src/core/types/api/user.ts

```diff
@@ -2,19 +2,28 @@ type BaseUserType = {
   ID: number;
   Username: string;
   IsAdmin: boolean;
-  RestrictedTags: Array<number>;
-  Avatar: {
-    RelativeFilepath: string;
-    ID: number;
-  };
+  RestrictedTags: number[];
```

Comment:
I can change back the `number[]` to `Array<number>` if you would rather have the full types for the primitive array values. @hidden4003

Reply:
Though personally I like to use the simplified array declarations when possible (meaning when the type in the array does not include any type parameters).